tests/v2/test_wakunode.nim

```diff
@@ -251,3 +254,16 @@ procSuite "WakuNode":
     await node1.stop()
     await node2.stop()
     await node3.stop()
+
+  asyncTest "Peer info parses correctly":
+    ## This is such an important utility function for wakunode2
+    ## that it deserves its own test :)
+    
+    let
+      addrStr = "/ip4/127.0.0.1/tcp/60002/p2p/16Uuu2HBmAcHvhLqQKwSSbX6BG5JLWUDRcaLVrehUVqpw7fz1hbYc"
```

Comment:
Just came to my mid, you may want to consider some corner cases, with wrong formats (in case such situation may happen), otherwise, it is fine

Reply:
Done :) 4b5094d